pmm/pmm2-ami-upgrade-tests.groovy

```diff
@@ -60,6 +60,9 @@ void setInstanceAMIId(PMM_VERSION) {
         case "2.21.0":
             env.AMI_ID = "ami-0605d9dbdc9d6a233"
             break;
+        case "2.22.0":
+            env.AMI_ID = "ami-0c1521c259375cb43"
```

Comment:
How are you sure about this? should we not wait until RC release day

Reply:
Do you mean 2.22.0 release day? We already have RC for 2.22.0 or do we add only released AMI here?

Reply:
I think it is fine to use the RC version, I do the same, we use same for release also, but what if we need to build a new RC?

Reply:
If we'll build a new RC we just replace ID here with a new PR :)